modules/social_features/social_group/social_group.install

```diff
@@ -640,3 +642,74 @@ function social_group_update_8906() {
     $config->save(TRUE);
   }
 }
+
+/**
+ * Helper function to import configurations.
+ *
+ * @param array $configs
+ *   Array of configuration file names and their folders.
+ */
+function _social_group_features_import_config_helper(array $configs) {
+  foreach ($configs as $folder => $config_files) {
+    foreach ($config_files as $config_file) {
+      $config = drupal_get_path('module', 'social_group') . "/config/{$folder}/{$config_file}.yml";
+
+      if (is_file($config)) {
+        $settings = Yaml::parse(file_get_contents($config));
+        if (is_array($settings)) {
+          $update_config = \Drupal::configFactory()
+            ->getEditable($config_file);
+
+          $update_config->setData($settings)->save(TRUE);
+        }
+      }
+    }
+  }
+}
+
+/**
+ * Import flag "Mute Group Notifications" configurations.
+ */
+function social_group_update_8907() {
+  $configs = [
+    'install' => [
+      'flag.flag.mute_group_notify',
+    ],
+  ];
+
+  _social_group_features_import_config_helper($configs);
+}
+
+/**
+ * Update the permissions for social_group.
+ */
+function social_group_update_8908() {
+  _social_group_set_permissions();
```

Comment:
Instead just add the required permissions to required roles, we don't to accidentally change the current permission changes.

Reply:
Check how it looks now.